minio/minio
# ui: Handle policies listed in mc as 'none' by not showing them

## Description

This commit handles policies listed in `mc` as `none` by not showing them on the policy list in the browser.

## Motivation and Context


Previously, the case where a policy was listed in `mc` as `none` wasn't handled at all as when a policy has been set as `none` it was removed from the list. However, it has been observed that this isn't always the case [here](https://github.com/minio/minio/issues/6349#issuecomment-415603086). For this behavior, the UI was defaulting back to and incorrectly displaying "Read Only" instead of showing that there is no policy, which was misleading the users.

## Regression


No

## How Has This Been Tested?



Manually and unit tests. The manual steps followed are as below:
1. Start from fresh FS disk
2. Login into Minio Browser
3. Create a bucket
4. Show bucket policy popup
5. Add 'Write Only' policy for '*'
6. Add 'Write Only' policy for 'foo'
7. Add 'Read Only' policy for 'foo'
8. Verify that the policy for '*' disappears and we only have the 'Read Only' policy for 'foo'

## Types of changes


## Checklist:
